Find a functionally equivalent code:android.transition.TransitionSet.resume:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * @hide
 */

@@ -1,7 +1,7 @@
 {
-    super.resume();
+    super.resume(sceneRoot);
     int numTransitions = mTransitions.size();
     for (int i = 0; i < numTransitions; ++i) {
-        mTransitions.get(i).resume();
+        mTransitions.get(i).resume(sceneRoot);
     }
 }

